services/src/main/java/org/keycloak/services/managers/AuthenticationManager.java

```diff
@@ -800,6 +800,21 @@ public static Response redirectAfterSuccessfulFlow(KeycloakSession session, Real
 
     }
 
+    public static String getSessionCookieValue(KeycloakSession session) {
```

Comment:
I would slightly prefer the name of this method to be "getSessionIdFromSessionCookie" as it is not returning "sessionCookieValue", but just the sessionId. This is minor thing, so I am approving, but if you have a chance to fix a bit, it will be nice :)